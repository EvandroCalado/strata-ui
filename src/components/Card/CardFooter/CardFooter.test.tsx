import { render } from '@testing-library/react';
import { CardFooter } from '.';

describe('<CardFooter />', () => {
  it('should render a snapshot', () => {
    const { container } = render(<CardFooter />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
