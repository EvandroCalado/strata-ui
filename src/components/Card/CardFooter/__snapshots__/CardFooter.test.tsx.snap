// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<CardFooter /> > should render a snapshot 1`] = `
<footer
  class=""
/>
`;
